cluster.ml

```diff
@@ -70,6 +70,7 @@ let deployment =
       ~zone:gcloud_zone
       ~java:`Oracle_7
       ~machine_type:(`Google_cloud `Highmem_8)
+      ~additional_packages:["postgresql-9.4"]
```

Comment:
I can merge yes.

I was thinking that the additional-packages should be in a `configuration.env` environment variable. That way other users can use the feature; and most users don't get a PG server running on every node for nothing.